CHANGELOG.md

```diff
@@ -6,7 +6,11 @@ Mapbox welcomes participation and contributions from everyone.
 #### Features
 
 #### Bug fixes and improvements
-
+- Added `Debug` logs to HistoryRecorder. [#5905](https://github.com/mapbox/mapbox-navigation-android/pull/5905)
+- Added explicitly message fail EV `Onboard` routing request. [#5905](https://github.com/mapbox/mapbox-navigation-android/pull/5905)
+- Fixed multiple switching to alternatives and incorrect switching to a close alternative route. [#5905](https://github.com/mapbox/mapbox-navigation-android/pull/5905)
```

Comment:
@DenisPryt , I'm not sure that Android SDK user will understand what was fixed. Does the suggested option make sense?
```suggestion
- Fixed an incorrect switch to an alternative route in case when the alternative route was too close to current position [#5905](https://github.com/mapbox/mapbox-navigation-android/pull/5905)
```
BTW, was this fix actual for Android or only for iOS reroute controller?